docs/manifest.json

```diff
@@ -83,6 +83,30 @@
 		"markdown_source": "https://raw.githubusercontent.com/WordPress/gutenberg/master/docs/extensibility/autocomplete.md",
 		"parent": "extensibility"
 	},
+	{
```

Comment:
This file is autogenerated, you need to update `root-manifest.json` file instead.

Reply:
Ah, once you update the proper file, you will need to run `npm run docs:build` to regenerate this file 🙃

Reply:
😄 Ah, that does it! Thanks @gziolo. Updated in https://github.com/WordPress/gutenberg/pull/8726/commits/03905f2735e9d3fd7a2cd926a6bdce7cee95612b